Frameworks/CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -363,10 +359,9 @@ CF_PRIVATE Boolean __CFProcessIsRestricted();
 
 // WINOBJC: The reference platform uses a special compiler option not available to us to 
 // get __CFConstantStringClassReferencePtr filled out with the correct "Class". Since this isn't

```

Comment:
Most of this comment is in CFBridgeUtilities? Confused which part you're asking for

---
In reply to: [78862194](https://github.com/Microsoft/WinObjC/pull/1008#discussion_r78862194) [](ancestors = 78862194)